yarelismartin/Bangthatzon-FE
# created the seller's products page

## Description
When a user if viewing the details of a product they should be able to click the sellers name and be directed to their seller page. This page will contain all the products the seller is selling.
## Related Issue
#1 




## Motivation and Context
This allows for a customer to navigate to their favorite seller and see what else they could buy from them.

## How Can This Be Tested?
Go to the home page
Click on the name of a product
When in the product detail page click the seller name
You should see all their products

## Screenshots (if appropriate):
## Types of changes

[ ] Bug fix (non-breaking change which fixes an issue)
[ x] New feature (non-breaking change which adds functionality)
[ ] Breaking change
